_data/analysis-files.yml

```diff
@@ -0,0 +1,36 @@
+- file: continent-estimated.csv
+  name: Estimated Failure Rates By Continent
+  desc: The estimated failure rates of data portals and journal websites grouped by continents
+- file: country-estimated.csv
```

Comment:
This is a good idea. Since this will require some structural change in the `data.md` code (i.e., iterating over "analysis-files" instead of "csv_files"), I will make it a separate PR.